ha/util/conf_store.py

```diff
@@ -221,3 +221,30 @@ def get_disk_list_for_cvg(index, node_id, cvg_id):
         except Exception as e:
             Log.error(f"Unable to fetch Disk list from GConf. Error {e}")
             raise Exception(f"Unable to fetch Disk list. Error {e}")
+
+    @staticmethod
+    def get_control_nodes(index):
+        """
+        Returns list of machine_ids for control-node.
+        Args:
+            index(str): index of conf file
+            parent key : node
+            search key: name
+            search val: csm
+        Returns:
+            list: list of node_ids
+
+        >>> Conf.search("cortx", 'node', 'name', 'csm')
+        ['node>a3710eee36644f2fb60ba19486664495>components[1]>name']
+        """
+        node_ids = []
+        try:
+            node_key = Conf.search(index, GconfKeys.NODE_CONST.value,
+                                   GconfKeys.NAME_CONST.value, Const.COMPONENT_CSM.value)
+            for key in node_key:
+                value = key.split('>')[1]
+                node_ids.append(value)
+            Log.info(f"Fetched control node machine-id: {node_ids}")
```

Comment:
Suggestion: Please avoid logging info level message in util.

Reply:
ACK